altair/jupyter/__init__.py

```diff
@@ -0,0 +1,17 @@
+try:
+    import anywidget  # noqa: F401
+    from .jupyter_chart import JupyterChart
```

Comment:
Could we move the `JupyterChart` import into an `else` block so that we don't suppress import errors happening within `jupyter_chart` module?

Something like:

```python
try:
    import anywidget
except ImportError:
    # Existing code
else:
    from .jupyter_chart import JupyterChart
```

Reply:
Done in 948a23d061fa0a4ac371937aa0d93a6f8d38b925